.github/mlc_config.json

```
@@ -0,0 +1,8 @@
+{
+    "replacementPatterns": [
+      {
+        "pattern": "^/tools",
+        "replacement": "https://nf-co.re/tools"
```

Cool!

Is it possible to generalise this even more..? eg:

```suggestion
        "pattern": "^/(*)",
        "replacement": "https://nf-co.re/$1"
```

(assuming it's possible to capture matches)

---

We could also go the other way and remove this file entirely, avoid using relative _website-only_ links completely and just use the full https://nf-co.re/tools URL wherever we need to for this particular page..

Might be easier in the long run as that should guarantee that links work everywhere 👍

---

Either is easy to do I guess. I can replace it everywhere with https://nf-co.re/tools also to keep it uniform.

---

@ewels the general pattern also works